_sass/color_schemes/memo.scss

```diff
@@ -1,3 +1,58 @@
 // Change default font family
 $body-font-family: "Inter", system-ui, -apple-system, blinkmacsystemfont,
   "Segoe UI", roboto, "Helvetica Neue", arial, sans-serif;
+
+  // Typography
+
+  $body-font-family: "Inter";
+  $root-font-size: 15px; // Base font-size for rems
+  $body-line-height: 1.5;
+  $content-line-height: 1.5;
+  $body-heading-line-height: 1.25;
+
+
+  $font-size-1: 11px;
+  $font-size-1-sm: 11px;
```

Comment:
What does `sm` stand for?

Reply:
> // `-sm` suffix is the size at the small (and above) media query

Source : https://github.com/just-the-docs/just-the-docs/blob/adbc25dec583e4d19f1511e53dce47785cfd00ec/_sass/support/_variables.scss#L12